source/includes/write/gridfs.php

```diff
@@ -0,0 +1,70 @@
+<?php
+require 'vendor/autoload.php'; // include Composer's autoloader
+
+use MongoDB\Client;
+use MongoDB\BSON\ObjectID;
+use MongoDB\GridFS\Bucket;
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+// Creates a GridFS bucket or references an existing one
+// start create bucket
+$db = $client->db;
+$bucket = $db->selectGridFSBucket();
+// end create bucket
+
+// Constructs a GridFS bucket
+// start create bucket explicit
+$manager = new Manager('<connection string>');
+$bucket = new Bucket($manager, 'db');
+// end create bucket explicit
+
+// Creates or references a GridFS bucket with a custom name
+// start create custom bucket
+$db = $client->db_gridfs;
```

Comment:
Q: is it necessary to show this line?

Reply:
I changed to $custom_bucket = $client->db->selectGridFSBucket(), so it still specifies the database